libr/util/str.c

```diff
@@ -1351,6 +1351,32 @@ R_API char *r_str_escape(const char *buf) {
 	return r_str_escape_ (buf, false, true, true, false, true);
 }
 
+R_API char *r_str_escape_r2(const char *buf) {
+	r_return_val_if_fail (buf, NULL);
+	char *new_buf = malloc (1 + strlen (buf) * 2);
+	if (!new_buf) {
+		return NULL;
+	}
+	const char *p = buf;
+	char *q = new_buf;
+	while (*p) {
+		switch (*p) {
+		case ';':
+		case '$':
+		case '`':
+		case '\\':
+		case '"':
+			*q++ = ' ';
+			p++;
+			/* FALLTHRU */
```

Comment:
Why fallthrough? Won't this make `pd 42 $$(!rm -rf /)` into `pd 42 $(!rm -rf /)` then?

Reply:
fixed dunno what i smoked